package.json

```diff
@@ -103,5 +104,7 @@
     "why-is-node-running": "^2.3.0",
     "yargs": "^17.7.2"
   },
-  "dependencies": {}
+  "dependencies": {
+    "@msgpack/msgpack": "^3.1.0"
```

Comment:
I _think_ we have to put this one in `devDependencies` because it'll be bundled in the library.